src/azul/indexer/__init__.py

```diff
@@ -243,7 +243,11 @@ def aggregate(self, tallies: Tallies):
             # Read all contributions from Elasticsearch
             contributions = self._read_contributions(total_tallies)
             actual_tallies = Counter(contribution.entity for contribution in contributions)
-            assert len(tallies) == len(actual_tallies)
+            # FIXME: move log here and use actual_tallies for it, as opposed to contributions
```

Comment:
I believe this is referencing the INFO and DEBUG logging of contributions in `_aggregate()`.

It should be noted that in the original location the logs reflect the culling of contributions ("For each entity and bundle, find the most recent contribution that is not a deletion") and in the new location the logs do not. I've made this change in a separate commit in case we want to revert it: https://github.com/DataBiosphere/azul/pull/1716/commits/99bbb8ea510564ec39c31c7a2ef8917f9a64663c


Here are a few examples when I ran a test with both the old and new logging in place:

```
# Example where logs were the same

2020-04-29 17:56:25,139 INFO    MainThread: Read 6 contribution(s). 
2020-04-29 17:56:25,139 DEBUG   MainThread: Number of contributions read, by entity: {'bundles/aaa96233-bf27-44c7-82df-b4dc15ad4d9d': 1, 'cell_suspensions/412898c5-5b9b-4907-b07c-e9b89666e204': 1, 'files/0c5ac7c0-817e-40d4-b1b1-34c3d5cfecdb': 1, 'files/70d1af4a-82c8-478a-8960-e9028b3616ca': 1, 'projects/e8642221-4c2c-4fd7-b926-a68bce363c88': 1, 'samples/a21dc760-a500-4236-bcff-da34a0e873d2': 1}
2020-04-29 17:56:25,139 INFO    MainThread: Selected 6 contribution(s) to be aggregated.
2020-04-29 17:56:25,139 DEBUG   MainThread: Number of contributions selected for aggregation, by entity (new): {'bundles/aaa96233-bf27-44c7-82df-b4dc15ad4d9d': 1, 'cell_suspensions/412898c5-5b9b-4907-b07c-e9b89666e204': 1, 'files/0c5ac7c0-817e-40d4-b1b1-34c3d5cfecdb': 1, 'files/70d1af4a-82c8-478a-8960-e9028b3616ca': 1, 'projects/e8642221-4c2c-4fd7-b926-a68bce363c88': 1, 'samples/a21dc760-a500-4236-bcff-da34a0e873d2': 1}
2020-04-29 17:56:25,139 INFO    MainThread: Selected 6 contribution(s) to be aggregated.
2020-04-29 17:56:25,139 DEBUG   MainThread: Number of contributions selected for aggregation, by entity (old): {'bundles/aaa96233-bf27-44c7-82df-b4dc15ad4d9d': 1, 'cell_suspensions/412898c5-5b9b-4907-b07c-e9b89666e204': 1, 'files/0c5ac7c0-817e-40d4-b1b1-34c3d5cfecdb': 1, 'files/70d1af4a-82c8-478a-8960-e9028b3616ca': 1, 'projects/e8642221-4c2c-4fd7-b926-a68bce363c88': 1, 'samples/a21dc760-a500-4236-bcff-da34a0e873d2': 1}


# Examples where logs were different:

2020-04-29 17:56:25,872 INFO    MainThread: Read 12 contribution(s). 
2020-04-29 17:56:25,872 DEBUG   MainThread: Number of contributions read, by entity: {'bundles/aaa96233-bf27-44c7-82df-b4dc15ad4d9d': 2, 'cell_suspensions/412898c5-5b9b-4907-b07c-e9b89666e204': 2, 'files/0c5ac7c0-817e-40d4-b1b1-34c3d5cfecdb': 2, 'files/70d1af4a-82c8-478a-8960-e9028b3616ca': 2, 'projects/e8642221-4c2c-4fd7-b926-a68bce363c88': 2, 'samples/a21dc760-a500-4236-bcff-da34a0e873d2': 2}
2020-04-29 17:56:25,872 INFO    MainThread: Selected 12 contribution(s) to be aggregated.
2020-04-29 17:56:25,872 DEBUG   MainThread: Number of contributions selected for aggregation, by entity (new): {'bundles/aaa96233-bf27-44c7-82df-b4dc15ad4d9d': 2, 'cell_suspensions/412898c5-5b9b-4907-b07c-e9b89666e204': 2, 'files/0c5ac7c0-817e-40d4-b1b1-34c3d5cfecdb': 2, 'files/70d1af4a-82c8-478a-8960-e9028b3616ca': 2, 'projects/e8642221-4c2c-4fd7-b926-a68bce363c88': 2, 'samples/a21dc760-a500-4236-bcff-da34a0e873d2': 2}
2020-04-29 17:56:25,872 INFO    MainThread: Selected 6 contribution(s) to be aggregated.
2020-04-29 17:56:25,872 DEBUG   MainThread: Number of contributions selected for aggregation, by entity (old): {'bundles/aaa96233-bf27-44c7-82df-b4dc15ad4d9d': 1, 'cell_suspensions/412898c5-5b9b-4907-b07c-e9b89666e204': 1, 'files/0c5ac7c0-817e-40d4-b1b1-34c3d5cfecdb': 1, 'files/70d1af4a-82c8-478a-8960-e9028b3616ca': 1, 'projects/e8642221-4c2c-4fd7-b926-a68bce363c88': 1, 'samples/a21dc760-a500-4236-bcff-da34a0e873d2': 1}

2020-04-29 17:56:26,437 INFO    MainThread: Read 18 contribution(s). 
2020-04-29 17:56:26,437 DEBUG   MainThread: Number of contributions read, by entity: {'bundles/aaa96233-bf27-44c7-82df-b4dc15ad4d9d': 3, 'cell_suspensions/412898c5-5b9b-4907-b07c-e9b89666e204': 3, 'files/0c5ac7c0-817e-40d4-b1b1-34c3d5cfecdb': 3, 'files/70d1af4a-82c8-478a-8960-e9028b3616ca': 3, 'projects/e8642221-4c2c-4fd7-b926-a68bce363c88': 3, 'samples/a21dc760-a500-4236-bcff-da34a0e873d2': 3}
2020-04-29 17:56:26,438 INFO    MainThread: Selected 18 contribution(s) to be aggregated.
2020-04-29 17:56:26,438 DEBUG   MainThread: Number of contributions selected for aggregation, by entity (new): {'bundles/aaa96233-bf27-44c7-82df-b4dc15ad4d9d': 3, 'cell_suspensions/412898c5-5b9b-4907-b07c-e9b89666e204': 3, 'files/0c5ac7c0-817e-40d4-b1b1-34c3d5cfecdb': 3, 'files/70d1af4a-82c8-478a-8960-e9028b3616ca': 3, 'projects/e8642221-4c2c-4fd7-b926-a68bce363c88': 3, 'samples/a21dc760-a500-4236-bcff-da34a0e873d2': 3}
2020-04-29 17:56:26,438 INFO    MainThread: Selected 6 contribution(s) to be aggregated.
2020-04-29 17:56:26,438 DEBUG   MainThread: Number of contributions selected for aggregation, by entity (old): {'bundles/aaa96233-bf27-44c7-82df-b4dc15ad4d9d': 1, 'cell_suspensions/412898c5-5b9b-4907-b07c-e9b89666e204': 1, 'files/0c5ac7c0-817e-40d4-b1b1-34c3d5cfecdb': 1, 'files/70d1af4a-82c8-478a-8960-e9028b3616ca': 1, 'projects/e8642221-4c2c-4fd7-b926-a68bce363c88': 1, 'samples/a21dc760-a500-4236-bcff-da34a0e873d2': 1}

2020-04-29 17:56:52,843 INFO    MainThread: Read 12 contribution(s). 
2020-04-29 17:56:52,843 DEBUG   MainThread: Number of contributions read, by entity: {'bundles/aaa96233-bf27-44c7-82df-b4dc15ad4d9d': 2, 'cell_suspensions/412898c5-5b9b-4907-b07c-e9b89666e204': 2, 'files/0c5ac7c0-817e-40d4-b1b1-34c3d5cfecdb': 2, 'files/70d1af4a-82c8-478a-8960-e9028b3616ca': 2, 'projects/e8642221-4c2c-4fd7-b926-a68bce363c88': 2, 'samples/a21dc760-a500-4236-bcff-da34a0e873d2': 2}
2020-04-29 17:56:52,843 INFO    MainThread: Selected 12 contribution(s) to be aggregated.
2020-04-29 17:56:52,843 DEBUG   MainThread: Number of contributions selected for aggregation, by entity (new): {'bundles/aaa96233-bf27-44c7-82df-b4dc15ad4d9d': 2, 'cell_suspensions/412898c5-5b9b-4907-b07c-e9b89666e204': 2, 'files/0c5ac7c0-817e-40d4-b1b1-34c3d5cfecdb': 2, 'files/70d1af4a-82c8-478a-8960-e9028b3616ca': 2, 'projects/e8642221-4c2c-4fd7-b926-a68bce363c88': 2, 'samples/a21dc760-a500-4236-bcff-da34a0e873d2': 2}
2020-04-29 17:56:52,844 INFO    MainThread: Selected 0 contribution(s) to be aggregated.
2020-04-29 17:56:52,844 DEBUG   MainThread: Number of contributions selected for aggregation, by entity (old): {}
```